packages/autorest.gotest/test/integrationtest/output/appplatform/armappplatform/apps_client_example_test.go

```diff
@@ -32,11 +33,41 @@ func ExampleAppsClient_Get() {
 	if err != nil {
 		log.Fatalf("failed to finish the request: %v", err)
 	}
-	// TODO: use response item
+	// TODO: use response
```

Comment:
should we remove the `TODO`?

Reply:
If we remove the TODO part, will it be weird that we use blank identifier to hold the res? @xboxeer What do you think?

Reply:
> should we remove the `TODO`?

@ArcturusZhang do you have any reason why we should remove the TODO, for me both having it or remove it looks good